bin/ringling-cli

```diff
@@ -16,19 +16,45 @@ limitations under the License.
 
 import argparse
 from ringling.test_script import print_test
+from ringling.projects import create_project, list_projects, get_project
```

Comment:
Change to one import per line.  Since git works on a per-line basis, it makes it clearer when an import is added or removed